src/fetching/postCampaign.ts

```diff
@@ -12,6 +12,6 @@ export default function postCampaign(orgId : string, fetch = defaultFetch) {
             method: 'POST',
         });
         const resData = await res.json();
-        return resData;
+        return resData?.data;
```

Comment:
We should make a note of this in some cleanup issue that we're not returning from the fetch functions consistently -- there are some fetch functions that will throw an error, and some like this, that will return undefined if there is no `resData`

Reply:
Good idea!

@omidhq Can you create an issue for making return values from fetch functions consistent?

Reply:
@richardolsson Sure!